<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Core::Internal::GeneralSettings</class>
 <widget class="QWidget" name="Core::Internal::GeneralSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>296</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <layout class="QFormLayout" name="formLayout">
      <property name="rowWrapPolicy">
       <enum>QFormLayout::WrapLongRows</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="colorLabel">
        <property name="text">
         <string>User &amp;interface color:</string>
        </property>
        <property name="buddy">
         <cstring>colorButton</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="colorHorizontalLayout">
        <item>
         <widget class="Utils::QtColorButton" name="colorButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>64</width>
            <height>0</height>
           </size>
          </property>
          <property name="alphaAllowed">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="resetButton">
          <property name="toolTip">
           <string>Reset to default</string>
          </property>
          <property name="text">
           <string>R</string>
          </property>
          <property name="icon">
           <iconset resource="core.qrc">
            <normaloff>:/core/images/reset.png</normaloff>:/core/images/reset.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="terminalLabel">
        <property name="text">
         <string>Terminal:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="terminalHorizontalLayout">
        <item>
         <widget class="QLineEdit" name="terminalEdit"/>
        </item>
        <item>
         <widget class="QToolButton" name="resetTerminalButton">
          <property name="toolTip">
           <string>Reset to default</string>
          </property>
          <property name="text">
           <string>R</string>
          </property>
          <property name="icon">
           <iconset resource="core.qrc">
            <normaloff>:/core/images/reset.png</normaloff>:/core/images/reset.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="editorLabel">
        <property name="text">
         <string>External editor:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="editorHorizontalLayout">
        <item>
         <widget class="QLineEdit" name="externalEditorEdit"/>
        </item>
        <item>
         <widget class="QToolButton" name="resetEditorButton">
          <property name="toolTip">
           <string>Reset to default</string>
          </property>
          <property name="text">
           <string>R</string>
          </property>
          <property name="icon">
           <iconset resource="core.qrc">
            <normaloff>:/core/images/reset.png</normaloff>:/core/images/reset.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="helpExternalEditorButton">
          <property name="text">
           <string>?</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="modifiedLabel">
        <property name="text">
         <string>When files are externally modified:</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="reloadBehavior">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>Always ask</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Reload all modified files</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ignore modifications</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>111</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::QtColorButton</class>
   <extends>QToolButton</extends>
   <header location="global">utils/qtcolorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="core.qrc"/>
 </resources>
 <connections/>
</ui>
